templates/default/rt/rtsearch.html

```diff
@@ -207,31 +194,35 @@ <H1>{$layout.pagetitle}</H1>
 		</TD>
 	</TR>
 	<TR>
-        <TD class="bold nobr"><img src="img/netdev.gif" alt=""> {trans("Device:")}</TD>
+        <TD class="bold nobr">{icon name="netdev" label="Device:" class="fa-fw"}</TD>
 		<TD class="nobr" id="rtnetdevs">
 			{include file="rt/rtnetdevs.html" form="search"}
 		</TD>
 	</TR>
-        <TR>
-		<TD class="bold nobr">{box_row icon="verifier" label="Verifier:" field_id="rtverifiers"}{/box_row}</TD>
+    <TR>
+		<TD class="bold nobr">
+			{icon name="verifier" label="Verifier:" field_id="rtverifiers"}
+		</TD>
 		<TD>
-                        <SELECT size="1" name="search[verifierid]" {tip text="Select user/owner of the ticket"}>
-                                <OPTION value="0">{trans("- all owners -")}</OPTION>
-                                {foreach $userlist as $item}
-                                <OPTION value="{$item.id}" {if $item.id == $search.verifierid}SELECTED{/if}>{$item.rname}</OPTION>
-                                {/foreach}
-                        </SELECT>
-		</TD>
-        </TR>
-        <TR>
-		<TD class="bold nobr"><img src="img/calendar.gif" alt=""> {trans("Deadline expired:")}</TD>
+			<SELECT size="1" name="search[verifierid]" {tip text="Select user/owner of the ticket"}>
+				<OPTION value="0">{trans("- all owners -")}</OPTION>
+				{foreach $userlist as $item}
+				<OPTION value="{$item.id}" {if $item.id == $search.verifierid}SELECTED{/if}>{$item.rname}</OPTION>
+				{/foreach}
+			</SELECT>
+		</TD>
+    </TR>
+    <TR>
+		<TD class="bold nobr">
+			{icon name="deadline" label="Deadline expired:" class="fa-fw"}
+		</TD>
 		<TD>
 			<INPUT TYPE="checkbox" NAME="search[expired]" VALUE="1" {tip text="Check if ticket is expired" trigger="expired"}>
 		</TD>
-        </TR>
+    </TR>
 	<TR>
 		<TD class="buttons" colspan="2">
-			<A href="javascript:document.search.submit();" accesskey="s"><IMG src="img/search.gif" alt=""> {trans("Search")}</A>
+			{button type="submit" label="Search" icon="search"}<A href="javascript:document.search.submit();" accesskey="s">
 		</TD>
```

Comment:
a to co? Coś sed w Debianie nie za dobrze działa? :)

Reply:
Nie wszystko da się zrobić niskim nakładem pracy brakujące buttony robiłem ręcznie. Poprawiłem to.